monkey/monkey_island/cc/ui/src/components/pages/ConfigurePage.js

```diff
@@ -18,6 +18,7 @@ import ConfigImportModal from '../configuration-components/ImportConfigModal';
 import applyUiSchemaManipulators from '../configuration-components/UISchemaManipulators.tsx';
 import HtmlFieldDescription from '../configuration-components/HtmlFieldDescription.js';
 import CONFIGURATION_TABS_PER_MODE from '../configuration-components/ConfigurationTabs.js';
+import {SCHEMA} from '../../services/configuration/config_schema.js';
 
 const CONFIG_URL = '/api/configuration/island';
```

Comment:
Do we still need this?

Reply:
To what you are referring ?

Reply:
CONFIG_URL

Reply:
Nope, i think there is a separate task for it..